### ISSUE TITLE
Feauture Request | Non-USD currencies support in CMarketItem

### ISSUE THREAD

**Issue by tuktun** [DoctorMcKay/node-steamcommunity#215]
Are you going to make non-usd currencies support in CMarketItem and getMarketItem? For some reasons, i need this. Sometimes "requests to buy" on Steam community market exists in other currencies, but not in USD and i need to get it.

**Comment by anthony-kenikh** [comment]
Non-usd currencies already supported, but not documented yet. If look at the source code [here](https://github.com/DoctorMcKay/node-steamcommunity/blob/master/classes/CMarketItem.js#L4), wee can see that, for example, method `getMarketItem` has a third optional parameter `currency `.

**Comment by sliterok** [comment]
@DoctorMcKay maybe you could update wiki page about `getMarketItem` method so that people would know about that functional? That would be really awesome!
